.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "ocrd_anybaseocr/pix2pixHD"]
-	path = ocrd_anybaseocr/pix2pixhd
-	url = https://github.com/NVIDIA/pix2pixHD
```

Comment:
I am against dropping the submodule. My PyPI release was just a [workaround](https://github.com/OCR-D/ocrd_all/issues/303#issuecomment-1076935476) for the (commented out) relative path dependency. It's still unclear whether the upstream developers approve of me taking the name and maintainer role. At any rate, it should be clear what [concrete version](https://github.com/NVIDIA/pix2pixHD/pull/297) of pix2pixhd this depends on.

Also, since ocrd_all by default does not clone recursively, this should not be necessary at all.